sdks/js/packages/core/react/components/organization/plans/pricing-column.tsx

```diff
@@ -267,7 +273,11 @@ export const PlanPricingColumn = ({
   const selectedIntervalPricing = plan.intervals[selectedInterval];
 
   const action: PlanChangeAction = useMemo(() => {
-    if (selectedIntervalPricing.planId === currentPlan?.planId) {
+    if (
+      selectedIntervalPricing?.planId === currentPlan?.planId ||
```

Comment:
Can we have a named function for this check? Something that represents what business logic this is?

Reply:
updated the code